src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -339,19 +345,37 @@ private void notifyUser(MediaRequestEntity mediaRequestEntity,
             if (notificationTemplateName.equals(NotificationApi.NotificationTemplate.ERROR_PROCESSING_AUDIO.toString())) {
 
                 List<DefendantEntity> defendantList = mediaRequestEntity.getHearing().getCourtCase().getDefendantList();
+
                 List<String> defendantNames = new ArrayList<>();
-                for (DefendantEntity defendant : defendantList) {
-                    defendantNames.add(defendant.getName());
+                String defendants = NO_DEFENDANTS;
+                if (!defendantList.isEmpty()) {
+                    for (DefendantEntity defendant : defendantList) {
+                        defendantNames.add(defendant.getName());
+
+                    }
+
+                    defendants = defendantNames.stream().collect(Collectors.joining(","));
```

Comment:
just remembered about getDefendantStringList on courtCaseEntity, which could be used here also.
So this could be:-

```java
String defendants = String.join(", ", mediaRequestEntity.getHearing().getCourtCase().getDefendantStringList());

if(StringUtils.isBlank(defendants)){
    defendants = NO_DEFENDANTS;
}
```

Reply:
getDefendantStringList implemented